lib/mixlib/install/options.rb

```diff
@@ -155,6 +160,16 @@ def resolve_platform_version_compatibility_mode!
         end
       end
 
+      def proxy_settings
```

Comment:
Hi, (not expert here). 
I was wondering where will the default come from?
From what I see here: https://github.com/chef/mixlib-install/blob/ee2a0b97ff941606e34d071257ac6f38c48ecb73/lib/mixlib/install/options.rb#L175 they're not inherited from anywhere.